src/common/util/copy-clipboard.ts

```diff
@@ -1,12 +1,17 @@
-export const copyToClipboard = (str) => {
+export const copyToClipboard = async (str) => {
```

Comment:
The functions that use this helper should not also await it, since it is async.

Reply:
Still trying to wrap my head around the async/await stuff, but could it be that it's a typo and you meant "no**w** also await"?

Reply:
yes